lang/main.json

```diff
@@ -213,6 +213,11 @@
         "muteParticipantButton": "Mute",
         "muteParticipantDialog": "Are you sure you want to mute this participant? You won't be able to unmute them, but they can unmute themselves at any time.",
         "muteParticipantTitle": "Mute this participant?",
+        "muteMultipleParticipantBody": "You won't be able to unmute them, but they can unmute themselves at any time.",
+        "muteMultipleParticipantButton": "Mute",
```

Comment:
Please do not duplicate already available strings. When reusing the already available keys if appropriate, you reuse more than 30 languages translations.

Reply:
Thanks - my intent was to keep texts separate in case they need to reworded later. Will change to reuse existing strings.